src/schemaCompare/schemaCompareWebViewController.ts

```diff
@@ -973,17 +977,24 @@ export class SchemaCompareWebViewController extends ReactWebviewPanelController<
         }
     }
 
-    private getActiveServersList(): { [connectionUri: string]: string } {
-        const activeServers: { [connectionUri: string]: string } = {};
+    private getActiveServersList(): {
+        [connectionUri: string]: { profileName: string; server: string };
+    } {
+        const activeServers: { [connectionUri: string]: { profileName: string; server: string } } =
+            {};
         let seenServerNames = new Set<string>();
 
         const activeConnections = this.connectionMgr.activeConnections;
         Object.keys(activeConnections).forEach((connectionUri) => {
-            let serverName = activeConnections[connectionUri].credentials.server;
+            let credentials = activeConnections[connectionUri]
+                .credentials as IConnectionDialogProfile;
 
-            if (!seenServerNames.has(serverName)) {
-                activeServers[connectionUri] = serverName;
-                seenServerNames.add(serverName);
+            if (!seenServerNames.has(credentials.server)) {
+                activeServers[connectionUri] = {
+                    profileName: credentials.profileName,
```

Comment:
What if credential doesn't have a profile name? I don't think all connection profiles have a display name.

Reply:
If the profile name isn't present, which is most often the case then it falls back to using the server name.

Reply:
Dang, looks like that doesn't (yet) handle the profileName part, so you'd need to do this for now:
`connection.profileName ? connection.profileName : getConnectionDisplayName(connection)`

but it's still better to use the centralized method, IMO